treeherder/intermittents_commenter/fetch.py

```diff
@@ -22,3 +22,9 @@ def fetch_testrun_matrix():
     testrun_matrix_url = f"{firefoxci_artefact_api_url}/test-info-testrun-matrix.json"
     response = requests.get(testrun_matrix_url, headers={"User-agent": "mach-test-info/1.0"})
     return response.json()
+
+
+def get_summary_groups():
+    url = "https://treeherder.mozilla.org/api/groupsummary/"
+    response = requests.get(url, headers={"User-agent": "mach-test-info/1.0"})
```

Comment:
I believe this only gets a single day, we will want an entire range of data.  Also this takes ~25 seconds to return, which is why there is an artifact that exists with this data already, and it has 30 days of history, so you can choose which days you want. 

The only problem is you will need to combine data from different days to summarize it.

Reply:
OK, thanks for that precisions.
Can you give me the url of the summary group artifact please? and how to access a particular day?

Reply:
test-run-info.json: https://searchfox.org/mozilla-central/source/testing/testinfo.py#400

there is code in there to get the data and squash it to combine days into a larger data structure:
https://searchfox.org/mozilla-central/source/testing/testinfo.py#397

as this is run counts split up by day, when looking at a week (7 days) you will need to squash data, otherwise you can pull a specific date (current day - 1) and get results.